scipy/integrate/setup.py

```diff
@@ -28,66 +34,73 @@ def configuration(parent_package='',top_path=None):
     quadpack_test_src = [join('tests','_test_multivariate.c')]
     odeint_banded_test_src = [join('tests', 'banded5x5.f')]
 
-    config.add_library('mach', sources=mach_src,
-                       config_fc={'noopt':(__file__,1)})
-    config.add_library('quadpack', sources=quadpack_src)
-    config.add_library('lsoda', sources=lsoda_src)
-    config.add_library('vode', sources=vode_src)
-    config.add_library('dop', sources=dop_src)
+    config.add_library('mach', sources=mach_src, config_fc={'noopt':(__file__,1)},
```

Comment:
some more `lint_diff.py` pickiness: two spaces missing on this line

Reply:
fixed